sections/featured-collection.liquid

```diff
@@ -109,6 +109,8 @@
               {%- assign placeholder_image = 'product-apparel-' | append: forloop.rindex -%}
               {% render 'card-product',
                 show_vendor: section.settings.show_vendor,
+                media_aspect_ratio: section.settings.image_ratio,
```

Comment:
This PR looks good, but the placeholders look oddly aligned when the image ratio is set to portrait. Can we center them vertically here? 

<img width="1029" alt="Screenshot 2023-07-12 at 11 12 10 AM" src="https://github.com/Shopify/dawn/assets/1202812/8b046e05-d830-432c-aceb-b8cfe8820f79">